platform/android/export/export_plugin.cpp

```diff
@@ -1542,18 +1542,32 @@ void EditorExportPlatformAndroid::_process_launcher_icons(const String &p_file_n
 String EditorExportPlatformAndroid::load_splash_refs(Ref<Image> &splash_image, Ref<Image> &splash_bg_color_image) {
 	bool scale_splash = GLOBAL_GET("application/boot_splash/fullsize");
 	bool apply_filter = GLOBAL_GET("application/boot_splash/use_filter");
+	bool show_splash_image = GLOBAL_GET("application/boot_splash/show_image");
 	String project_splash_path = GLOBAL_GET("application/boot_splash/image");
 
-	if (!project_splash_path.is_empty()) {
-		splash_image.instantiate();
-		print_verbose("Loading splash image: " + project_splash_path);
-		const Error err = ImageLoader::load_image(project_splash_path, splash_image);
-		if (err) {
-			if (OS::get_singleton()->is_stdout_verbose()) {
-				print_error("- unable to load splash image from " + project_splash_path + " (" + itos(err) + ")");
+	// Setup the splash bg color
+	bool bg_color_valid;
```

Comment:
Needs to be initialized.

Reply:
Though maybe it can even be done without a boolean, using `get_setting("application/boot_splash/bg_color", boot_splash_bg_color)`.

Reply:
> Though maybe it can even be done without a boolean, using `get_setting("application/boot_splash/bg_color", boot_splash_bg_color)`.

It was already there, I wasn't sure what it did, I just changed its position

Reply:
That's okay for now 👍  But we should probably fix it at some point.